projects/step-frontend/src/lib/modules/execution/execution.module.ts

```diff
@@ -267,6 +281,16 @@ import { ExecutionAgentsListComponent } from './components/execution-agents-list
     AltExecutionTimePrefixDirective,
     AltExecutionTimeSuffixDirective,
     AltExecutionTimePopoverAddonDirective,
+    DatePipe,
+    FormsModule,
+    MatTooltip,
+    AsyncPipe,
+    AsyncPipe,
+    MatMenu,
+    MatMenu,
+    MatMenuContent,
+    MatMenuContent,
+    MatMenuTrigger,
```

Comment:
These imports don't look correct.
All of them are already come from `StepCoreModule`, which as re-exported dependency of `StepCommonModule`